src/main/java/com/openlattice/postgres/PostgresTable.java

```diff
@@ -31,7 +32,7 @@
  * Tables definitions for all tables used in the OpenLattice platform.
  */
 public final class PostgresTable {
-
+    public static final PostgresTableDefinition JOBS = PostgresJobsMapStore.JOBS;
```

Comment:
What's this here for? Can't any consumer of `PostgresTable` consume `PostgresJobsMapStore` instead? Not a big deal in any case, just curious

Reply:
If the table isn't in this class the PostgresTables class won't be able to automatically pick it up for PostgresTableManager.

Reply:
Ahh okay, I didn't realize PostgresTable gets scanned. Cool 👍